scripts/manual_validation_results.qmd

```diff
@@ -0,0 +1,917 @@
+    has_summary_results_reg1_main == TRUE & has_summary_results_reg2_main == TRUE ~ "Both",
+    has_summary_results_reg1_main == FALSE & has_summary_results_reg2_main == TRUE ~ "Only other registry"),
+    result_label = factor(result_label, levels = c("Only other registry", "Both", "Only EUCTR", "No results"))) |>
+  group_by(result_label) |>
+  summarise(count = n(), .groups = 'drop') |>
+  mutate(percentage = count / sum(count) * 100) |>
+  mutate(label = paste0(count, " (", round(percentage, 0), "%)"))
+
+#ClinicalTrials.gov
+discrepancy_data |>
+  filter(overall_recruitment_status_reg1 == "Completed",
+         overall_recruitment_status_reg2 == "Completed",
+         registry2 == "ClinicalTrials.gov") |>
+  mutate(result_label = case_when(
+    has_summary_results_reg1_main == FALSE & has_summary_results_reg2_main == FALSE ~ "No results",
+    has_summary_results_reg1_main == TRUE & has_summary_results_reg2_main == FALSE ~ "Only EUCTR",
+    has_summary_results_reg1_main == TRUE & has_summary_results_reg2_main == TRUE ~ "Both",
+    has_summary_results_reg1_main == FALSE & has_summary_results_reg2_main == TRUE ~ "Only other registry"),
+    result_label = factor(result_label, levels = c("Only other registry", "Both", "Only EUCTR", "No results"))) |>
+  group_by(result_label) |>
+  summarise(count = n(), .groups = 'drop') |>
+  mutate(percentage = count / sum(count) * 100) |>
+  mutate(label = paste0(count, " (", round(percentage, 0), "%)"))
+
+#DRKS
+discrepancy_data |>
+  filter(overall_recruitment_status_reg1 == "Completed",
+         overall_recruitment_status_reg2 == "Completed",
+         registry2 == "DRKS") |>
+  mutate(result_label = case_when(
+    has_summary_results_reg1_main == FALSE & has_summary_results_reg2_main == FALSE ~ "No results",
+    has_summary_results_reg1_main == TRUE & has_summary_results_reg2_main == FALSE ~ "Only EUCTR",
+    has_summary_results_reg1_main == TRUE & has_summary_results_reg2_main == TRUE ~ "Both",
+    has_summary_results_reg1_main == FALSE & has_summary_results_reg2_main == TRUE ~ "Only other registry"),
+    result_label = factor(result_label, levels = c("Only other registry", "Both", "Only EUCTR", "No results"))) |>
+  group_by(result_label) |>
+  summarise(count = n(), .groups = 'drop') |>
+  mutate(percentage = count / sum(count) * 100) |>
+  mutate(label = paste0(count, " (", round(percentage, 0), "%)"))
+```
+
+### First sensitivity analysis
+
+For the sensitivity analysis, summary results were considered as reported (i.e. `has_summary_results_reg1_sensitivity == TRUE` or `has_summary_results_reg2_sensitivity == TRUE`) if any of the following were available in the summary results section:
+
+-   Structured tabular result
+
+-   Link to a file that reported results (such as synopsis, study reports, etc.)
+
+-   Link to a termination statement
+
+-   Link to a publication citation
+
+-   Link to publication or abstract
```

Comment:
@elisabascunan could you add also "Tabular results from another registry", if you came across at least one case?